guides/doc-Release_Notes/topics/katello.adoc

```diff
@@ -4,8 +4,42 @@
 [id="katello-headline-features"]
 == Headline Features
 
+=== Verify Content Checksum on container repositories using Hammer
+Added option '--validate-contents true/false' to "hammer repository synchronize" command.
+
+=== Add ACS bulk refresh option
+Added option '--all' to "hammer alternate-content-source refresh" command.
+ACS bulk refresh through API silently sanitizes input ids.
+
+=== Experimental activation key details page
+When the setting 'Show Experimental Labs' is enabled, the Activation Key 
+details page will be replaced by the beginnings of a redesigned page.
+
+=== Content Views and CVV enhancements to filters and refresh
+Content View filter creation/edit button now says 'Save' instead of 'Edit Rule'.
+Content view dependency solving tracked for needs_publish.
+
+=== Pulpcore and plugins updated from Pulpcore 3.22 to 3.28
+Contains both fixes and features that Katello can use in the future to better manage content, such as:
+https://github.com/pulp/pulp_rpm/issues/3135[More filter options on the packages API]
+https://github.com/pulp/pulp_rpm/issues/3009[New setting for generated primary.xml timestamps to match upstream repository]
+https://github.com/pulp/pulp_rpm/issues/2995[RPM replicator support]
+https://github.com/pulp/pulpcore/issues/3538[New distribution hiding option]
+https://github.com/pulp/pulpcore/issues/3835[OpenTelemetry support]
+https://github.com/pulp/pulpcore/issues/3656[File size added to content application listing]
```

Comment:
This now renders as one long line. Did you intend to make it a list?

Reply:
Yeah it should be a list... does it need newlines in between them?

Reply:
Is `\` the proper newline delimiter? 
Do we have a new rendered preview @ekohl ?